conf/default.toml

```diff
@@ -9,4 +9,4 @@
 # The advanced search filters available to the user can be customized with this setting.
 # The following fields can be used for filtering: 'organization', 'format', 'publicationYear', 'inspireKeyword', 'keyword', 'topic', 'isSpatial', 'license', 'resourceType', 'representationType', 'revisionYear', 'categoryKeyword', 'qualityScore', 'territories', 'publisherOrg', 'producerOrg'
 # any other field will be ignored
-# advanced_filters = ['producerOrg', 'categoryKeyword', 'revisionYear', 'license', 'qualityScore']
+# advanced_filters = ['producerOrg', 'categoryKeyword', 'territories', 'revisionYear', 'license', 'qualityScore']
```

Comment:
I think it'd even make sense to add the filter to the [default filters in the code](https://github.com/camptocamp/mel-dataplatform/blob/GSMEL-424-territories-search/apps/datahub/src/app/search/search-filters/search-filters.component.ts#L34), as the badge now has a behavior that depends on this filter. In this case, you would probably not need the `default.toml` fixture for the e2e tests.

Reply:
done!